src/components/executive-education-2u/utils.js

```diff
@@ -40,4 +42,18 @@ const getExecutiveEducationCoursePrice = (courseData) => {
   return {};
 };
 
-export { getCourseOrganizationDetails, getExecutiveEducationCoursePrice };
+const getCourseStartDate = (contentMetadata, activeCourseRun) => {
```

Comment:
[consideration/suggestion] I wonder if it might be worth being a bit more intentional about when to pull from `additionalMetadata` vs. the active course run. For example, might it make sense to extend the `COURSE_TYPE_CONFIG` setting defined in the MFE runtime config API ([source](https://github.com/edx/edx-internal/blob/326425bb0b2143335d73c84f5ba69246a4036971/edx-remote-config/prod/lms.yml#L1567-L1570)) to indicate that courses with `course_type: 'executive-education-2u'` should pull from additional metadata?

```yml
COURSE_TYPE_CONFIG:
      executive-education-2u:
        pathSlug: executive-education-2u
        usesEntitlementListPrice: true
        usesAdditionalMetadata: true
```

```jsx
import { getCourseTypeConfig } from 'path/to/src/components/course/data/utils';

const getCourseStartDate = ({ contentMetadata, activeCourseRun }) => {
  let startDate;
  const courseTypeConfig = getCourseTypeConfig(contentMetadata);

  if (courseTypeConfig?.usesAdditionalMetadata && contentMetadata?.additionalMetadata) {
    startDate = contentMetadata.additionalMetadata.startDate;
  } else {
    startDate = activeCourseRun?.start;
  }

  if (!startDate) {
    return undefined;
  }

  return moment(startDate).format(DATE_FORMAT);
};
```